server/src/main/java/com/witness/server/repository/ExerciseLogRepository.java

```diff
@@ -26,7 +26,7 @@ public interface ExerciseLogRepository extends JpaRepository<ExerciseLog, Long>
         FROM
           ExerciseLog e
         WHERE
-          e.exercise.id = :exerciseId AND e.workoutLog.user.id = :userId AND e.setLogs.size > 0
+          e.exercise.id = :exerciseId AND e.workoutLog.user.id = :userId AND size(e.setLogs) > 0
```

Comment:
Couldn't find much info on this. The `.size` syntax made the application fail during startup. Hibernate complained about an illegal dereference of the `List<SetLog>` using property `size`. Maybe that notation has been deprecated for this use case.